src/OpenLoco/src/Scenario.cpp

```diff
@@ -499,7 +499,7 @@ namespace OpenLoco::Scenario
     static loco_global<ObjectManager::SelectedObjectsFlags*, 0x50D144> _inUseobjectSelection;
     static loco_global<ObjectManager::ObjectSelectionMeta, 0x0112C1C5> _objectSelectionMeta;
 
-    static std::span<ObjectManager::SelectedObjectsFlags> getInUseSelectedObjectFlags()
+    std::span<ObjectManager::SelectedObjectsFlags> getInUseSelectedObjectFlags()
```

Comment:
NB: a similar (if not identical) function exists in `Title.cpp`. Should this one be used there as well?

Reply:
I plan (not done yet) to make all users of these flags to own a unique_ptr of the flags. That is why there are multiple versions of this same function.

Reply:
Oh, good to know. Should I make another instance of the function as well then?

Reply:
I've gone ahead and done just that. We can revert 85fb9a1 if you prefer otherwise.